### PR TITLE
docs: minor changes to improve documentation

### PR DESCRIPTION
**Pull request by nitinja**
Simple PR to fix few minor typos to improve docs quality 

- corrected few typos in .md files
- corrected typo in code sample at one place
- replaced "NGRX" with "Ngrx" to make it consistent like other places in the documentation

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngxs/store/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
Few typos are currently present in the documentation
Issue Number: N/A

## What is the new behavior?
PR fixes those typos

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@
 NGXS is a state management pattern + library for Angular. It acts as a single source of
 truth for your application's state, providing simple rules for predictable state mutations.
 
-NGXS is modeled after the CQRS pattern popularly implemented in libraries like Redux and NGRX
+NGXS is modeled after the CQRS pattern popularly implemented in libraries like Redux and Ngrx
```
Comment (arturovt) [inline docs/README.md]: `NgRx` then

File: docs/concepts/actions.md [hunk 2]
```diff
@@ -76,4 +76,4 @@ Examples:
 * [Project Stars Component] StarsUpdated
 
 A great video on the topic is [Good Action Hygiene by Mike Ryan](https://www.youtube.com/watch?v=JmnsEvoy-gY)
-It's for NGRX, but the same naming conventions apply to NGXS.
+It's for Ngrx, but the same naming conventions apply to NGXS.
```
Comment (arturovt) [inline docs/concepts/actions.md]: `NgRx` then

File: docs/README.md (hunk 1 above)
Comment (nitinja) [reply docs/README.md]: Changes every non-link occurrence of "Ngrx" to "NgRx", thanks for pointing that out.

File: docs/concepts/actions.md (hunk 2 above)
Comment (nitinja) [reply docs/concepts/actions.md]: done, with this change at some some additional places

**Pull request merged**